<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionCalibration</class>
 <widget class="QWidget" name="OptionCalibration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/icons/general/fondo_transparente.png);
border-image: url(:/icons/general/fondo_transparente.png);</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>359</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/icons/main_menu/calibration_menu/background_calibration_menu.png);</string>
   </property>
   <widget class="MyLabelAnimated" name="l_calibration_back">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>25</y>
      <width>320</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/main_menu/calibration_menu/header_calibration_menu.png);
background-image: url(:/icons/general/fondo_transparente.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MyLabelAnimated" name="l_calibration_state_button">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>163</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/icons/general/fondo_transparente.png);
border-image: url(:/icons/general/next_button.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="pt_calibration_state_text">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>281</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Swis721 Lt BT</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit { 
	background-image: url(:/icons/general/fondo_transparente.png);
	color: rgb(28, 28, 28);
	border-radius: 3px;
	text-align: center;
}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Swis721 Lt BT'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;El dispositivo deberá realizar una medición máxima y mínima. Para ello siga las instruciones durante el progreso una vez iniciado&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_calibration_state_footer_text_1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Swis721 Lt BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/icons/general/fondo_transparente.png);
color:rgb(62, 167, 132);</string>
    </property>
    <property name="text">
     <string>Última calibración</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="l_last_calibration_time">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>320</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Swis721 Lt BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/icons/general/fondo_transparente.png);
color:rgb(62, 167, 132);</string>
    </property>
    <property name="text">
     <string>12 dic. 2020 17:52</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="l_calibration_state">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Swis721 Lt BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/icons/general/fondo_transparente.png);
color:rgb(62, 167, 132);</string>
    </property>
    <property name="text">
     <string>Información</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="l_calibration_state_progress_text">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>280</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Swis721 Lt BT</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/icons/general/fondo_transparente.png);
color:rgb(62, 167, 132);</string>
    </property>
    <property name="text">
     <string>Medición mínima</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="pbar_calibration_state_progress">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>280</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Swis721 BT</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> QProgressBar::chunk {
	color: rgb(77, 77, 77);
     border-radius: 4px;
     background-color: #3ea784;
 }
QProgressBar {
	 color: rgb(77, 77, 77);
     border-radius: 4px;
     background-color: #E3E3E3;
 }</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="MyLabelAnimated" name="l_calibration_state_button_out">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>163</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/icons/general/fondo_transparente.png);
border-image: url(:/icons/general/exit_button.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="MyLabelAnimated" name="l_calibration_state_button_repeat">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>163</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/icons/general/fondo_transparente.png);
border-image: url(:/icons/general/repeat_button.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="l_calibration_state_error_tag">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>280</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Swis721 Lt BT</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/icons/general/fondo_transparente.png);
color:rgb(255, 50, 50);</string>
    </property>
    <property name="text">
     <string>Incompleta</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="l_calibration_state_error_text">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>280</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Swis721 Lt BT</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/icons/general/fondo_transparente.png);
color:rgb(255, 50, 50);</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLabelAnimated</class>
   <extends>QLabel</extends>
   <header location="global">mylabelanimated.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
